docs/quickstart/install.md

```diff
@@ -84,17 +78,32 @@ EOF
 sudo yum install tanzu-cli # dnf install can also be used
 ```
 
+### Note on installation paths
+
+Package managers have opinions on locations to which binaries are installed.
+If the location of the CLI (e.g `/opt/homebrew/bin/tanzu` by HomeBrew,
```

Comment:
FYI, although I think it doesn't matter for the doc, `/opt/homebrew` is the prefix for Apple M1 machines.  For intel Mac's it is something under `/usr...`.  That is why documentation often says, e.g. `$(brew --prefix)/bin/tanzu`

Reply:
Thanks, nice catch, I will update the doc as suggested.